app/waf/src/models/application.model.ts

```diff
@@ -0,0 +1,51 @@
+import {Entity, model, property} from '@loopback/repository';
+
+@model()
+export class Application extends Entity {
+  @property({
+    type: 'number',
```

Comment:
Id should be string type. We will need to generate string type uuids for them.

Reply:
Agree!